Library/Homebrew/linkage_checker.rb

```diff
@@ -247,7 +264,7 @@ def harmless_broken_link?(dylib)
   # Display a list of things.
   # Things may either be an array, or a hash of (label -> array)
   def display_items(label, things, puts_output: true)
-    return if things.empty?
+    return unless things.present?
```

Comment:
I feel like I have slight preference for the former. Why change it?

Reply:
I was getting an error here when `broken_dylibs_with_expectations` was `nil`. Let me check if this is still the case.

Reply:
Looks like it's no longer the case. I'll revert this change.